package.json

```diff
@@ -29,9 +29,9 @@
     "lint:flow": "flow --color always",
     "lint:build": "npm run lint && npm run build:cjs -s",
     "lint:build:test": "npm run lint:build && npm run test:coverage",
-    "test": "clear && jest",
-    "test:watch": "clear && jest --watch",
-    "test:coverage": "jest --coverage ",
+    "test": "clear && BABEL_DISABLE_CACHE=1 NODE_ENV=test jest --config jest.config.json",
```

Comment:
Good point. I usually put it in for tests to avoid getting cached transpilations, but i'll remove it until it actually becomes a problem.